QuestionBot/QuestionBot.UnitTests/Model/MessageEmitterTests.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using System.Collections.Generic;
+using NUnit.Framework;
+using QuestionBot.Model;
+using Moq;
+
+namespace QuestionBot.UnitTests.Model {
+    [TestFixture]
+    internal class MessageEmitterTests {
+        private MessageEmitter _testEmitter;
+        private Mock<IMessageListener> _testListener;
+        private Mock<IConsole> _testConsole;
+        private string _exitCommand;
+
+
+        [SetUp]
```

Comment:
Learning opportunity! (The test file is fine the way it's set up now but I figured I'd mention something fancy you can do)

As you already know, the function with the `[SetUp]` attribute gets called before every test. If you look at your 3 tests, you'll notice they pretty much all start with the same few commands. You can extract this common functionality and put it in this `Setup` method to make your code more readable.